project/Merging.scala

```diff
@@ -2,7 +2,7 @@ import sbtassembly.{MergeStrategy, PathList}
 
 object Merging {
   def customMergeStrategy(oldStrategy: (String) => MergeStrategy):(String => MergeStrategy) = {
-    case PathList("org", "joda", "time", "base", "BaseDateTime.class") => MergeStrategy.first
+    case x if x.endsWith("io.netty.versions.properties") => MergeStrategy.discard
```

Comment:
in ES 2.4, we had to ignore joda classes in `sbt assembly` because ES bundled them. Now, we have to deal with some netty conflicts.